### PR TITLE
Contact Form

### PR DESCRIPTION
**Pull request by niicode**
-Added a contact form to the footer of the website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -214,6 +214,37 @@ <h3 class="font-style">Skills</h3>
...
+                                placeholder="Your email" required data-field="email">
+                        </div>
+                        <div class="form-group">
+                            <textarea name="message" rows="10" cols="30" id="message" class="form-input font-style"
+                                placeholder="Write your message here" maxlength="500" required
+                                data-field="message"></textarea>
+                        </div>
+                        <div class="form-group web-centered">
+                            <button type="submit" class="button button-enabled font-style contact-button">Get In
+                                Touch</button>
```
Comment (codecaiine) [inline index.html:242]: - [ ] Good job so far with your contact form it's working fine :100: :+1: , however, your Button interactions specially for the `hover` need to be styled as Figma template, stick to the design as much as possible. :smiley: 

Screenshot from 2022-07-05 13-39-08

**Review by codecaiine — CHANGES_REQUESTED** [review]

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @niicode @Blossom-Aniedi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good use of Github flow :+1: 
- Good commit history :+1: 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (niicode) [reply index.html:242]: @codecaiine Please was there a specified color for the button on the contact form for `hover`

**Review by ShahierNashaat — APPROVED** [review]
Hi @niicode and @Blossom-Aniedi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**